src/ActionCreator.js

```diff
@@ -6,9 +8,30 @@ let makeDeleteAction = (snap) => {
 
 // Given an EBS volume and a list of snapshots, return a list of actions
 // to create the necessary snapshots that fulful the backup requirements
-let makeCreationActions = (volume, snapList) => {
+let makeCreationActions = (volume) => {
+	let backupTypes = volume.BackupConfig.BackupTypes.filter(type => {
+		let now = moment();
```

Comment:
Would the app be inconsistent if it run slowly? IE, if you're constantly checking the current time, and it's a second later than when you last checked it, would things break?